Gordon360/ApiControllers/JobsController.cs

```diff
@@ -0,0 +1,223 @@
+                }
+                result = _jobsService.saveShiftForUser(userID, shiftDetails.EML, shiftDetails.SHIFT_START_DATETIME, shiftDetails.SHIFT_END_DATETIME, shiftDetails.HOURS_WORKED, shiftDetails.SHIFT_NOTES, username);
+            }
+            catch (Exception e)
+            {
+                System.Diagnostics.Debug.WriteLine(e.Message);
+                return Request.CreateResponse(HttpStatusCode.InternalServerError, e);
+            }
+            return Request.CreateResponse(HttpStatusCode.OK, result);
+        }
+
+        /// <summary>
+        /// Get a user's active jobs
+        /// </summary>
+        /// <returns>The result of deleting the shift</returns>
+        [HttpDelete]
+        [Route("deleteShift/{rowID}")]
+        [StateYourBusiness(operation = Operation.DELETE, resource = Resource.SHIFT)]
+        public IHttpActionResult deleteShiftForUser(int rowID)
+        {
+            IEnumerable<StudentTimesheetsViewModel> result = null;
+            int userID = -1;
+
+            var authenticatedUser = this.ActionContext.RequestContext.Principal as ClaimsPrincipal;
+            var username = authenticatedUser.Claims.FirstOrDefault(x => x.Type == "user_name").Value;
+            var id = _accountService.GetAccountByUsername(username).GordonID;
+            userID = Convert.ToInt32(id);
+
+            try
+            {
+                result = _jobsService.deleteShiftForUser(rowID, userID);
+            }
+            catch (Exception e)
+            {
+                System.Diagnostics.Debug.WriteLine(e.Message);
+                return InternalServerError();
+            }
+            return Ok(result);
+        }
+
+        /// <summary>
+        /// Submit shifts
+        /// </summary>
+        /// <returns>The result of deleting the shift</returns>
+        [HttpPost]
+        [Route("submitShifts")]
+        [StateYourBusiness(operation = Operation.UPDATE, resource = Resource.SHIFT)]
+        public IHttpActionResult submitShiftsForUser([FromBody] IEnumerable<ShiftToSubmitViewModel> shifts)
+        {
+            IEnumerable<StudentTimesheetsViewModel> result = null;
+
+            try
+            {
+                foreach (ShiftToSubmitViewModel shift in shifts)
+                {
+                    System.Diagnostics.Debug.WriteLine("Submitting shift to " + shift.SUBMITTED_TO);
+                    result = _jobsService.submitShiftForUser(shift.ID_NUM, shift.EML, shift.SHIFT_END_DATETIME, shift.SUBMITTED_TO, shift.LAST_CHANGED_BY);
+                }
+            }
+            catch (Exception e)
+            {
+                System.Diagnostics.Debug.WriteLine(e.Message);
+                return InternalServerError();
+            }
+            return Ok(result);
+        }
+
+        /// <summary>
+        /// Submit shifts
+        /// </summary>
+        /// <returns>The result of deleting the shift</returns>
```

Comment:
Did you forget to change some of these comments?- more than one says "the result of deleting the shift"

Reply:
I definitely have a branch somewhere in which I changed these comments, I have no idea why they didn't make it into this one. I'll get on that lol

Reply:
sounds good